# Add icons for changelog and todo files

diff --git a/src/output/icons.rs b/src/output/icons.rs
index 1c64ca77..48024abe 100644
--- a/src/output/icons.rs
+++ b/src/output/icons.rs
@@ -112,6 +112,7 @@ impl Icons {
     const MAKE: char            = '\u{e673}';  // 
     const MARKDOWN: char        = '\u{f48a}';  // 
     const MUSTACHE: char        = '\u{e60f}';  // 
+    const NEWS: char            = '\u{f1ea}';  // 
     const NODEJS: char          = '\u{e718}';  // 
     const NOTEBOOK: char          = '\u{e678}';  // 
     const NPM: char             = '\u{e71e}';  // 
@@ -142,6 +143,7 @@ impl Icons {
     const TCL: char             = '\u{f06d3}'; // 󰛓
     const TERRAFORM: char       = '\u{f1062}'; // 󱁢
     const TEXT: char            = '\u{f15c}';  // 
+    const TODO: char            = '\u{f0ae}';  // 
     const TYPST: char           = '\u{f37f}';  // 
     const TMUX: char            = '\u{ebc8}';  // 
     const TOML: char            = '\u{e6b2}';  // 
@@ -306,6 +308,8 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     "Cargo.lock"          => Icons::LANG_RUST,      // 
     "Cargo.toml"          => Icons::LANG_RUST,      // 
     "CMakeLists.txt"      => '\u{e794}',            // 
+    "CHANGES"             => Icons::NEWS,           // 
+    "CHANGES.md"          => Icons::NEWS,           // 
     "CODE_OF_CONDUCT"     => '\u{f4ae}',            // 
     "CODE_OF_CONDUCT.md"  => '\u{f4ae}',            // 
     "composer.json"       => Icons::LANG_PHP,       // 
@@ -413,6 +417,8 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     "MANIFEST.in"         => Icons::LANG_PYTHON,    // 
     "mix.lock"            => Icons::LANG_ELIXIR,    // 
     "mpv.conf"            => '\u{f36e}',            // 
+    "NEWS"                => Icons::NEWS,           // 
+    "NEWS.md"             => Icons::NEWS,           // 
     "npm-shrinkwrap.json" => Icons::NPM,            // 
     "npmrc"               => Icons::NPM,            // 
     "package-lock.json"   => Icons::NPM,            // 
@@ -451,6 +457,8 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     "timezone"            => Icons::CLOCK,          // 
     "tmux.conf"           => Icons::TMUX,           // 
     "tmux.conf.local"     => Icons::TMUX,           // 
+    "TODO"                => Icons::TODO,           // 
+    "TODO.md"             => Icons::TODO,           // 
     "tsconfig.json"       => Icons::LANG_TYPESCRIPT,// 
     "Vagrantfile"         => '\u{2371}',            // ⍱
     "vlcrc"               => '\u{f057c}',           // 󰕼
